src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -84,27 +85,22 @@ def get(self, request: Request, project: Project, event_id: str) -> Response:
         if event is None:
             raise NotFound(detail="Event not found")
 
-        release_version = event.get_tag("sentry:release")
+        release_response = self._extract_release(event, project)
+        if type(release_response) is Response:
+            return release_response
+        else:
+            release = release_response
 
         exception = event.interfaces["exception"].values[int(exception_idx)]
 
         filename, abs_path = self._get_filename_and_path(exception, frame_idx)
 
-        if not release_version:
-            return self._create_response(issue=SourceMapProcessingIssue.MISSING_RELEASE)
-
-        try:
-            release = Release.objects.get(
-                organization=project.organization, version=release_version
-            )
-        except Release.DoesNotExist:
-            return self._create_response(issue=SourceMapProcessingIssue.MISSING_RELEASE)
-
         user_agent = release.user_agent
 
         if not user_agent:
             return self._create_response(
-                issue=SourceMapProcessingIssue.MISSING_USER_AGENT, data={"version": release_version}
+                issue=SourceMapProcessingIssue.MISSING_USER_AGENT,
+                data={"version": release.version, "fileName": filename},
```

Comment:
I don't recall ever seeing `fileName` capitalized like this in our codebase 😅

Reply:
fixed!